libbeat/docs/breaking.asciidoc

```diff
@@ -0,0 +1,105 @@
+[[breaking-changes]]
+== Breaking changes
+
+As a general rule, we strive to keep backwards compatibility between minor
+versions (e.g.  5.x to 5.y) so you can upgrade without any configuration file
+changes, but there are breaking changes between major versions (e.g. 5.x to
+6.y).
+
+We list the breaking changes at the top of the <<release-notes,release notes>>
+for each version.
+
+[[breaking-changes-5.0]]
+=== Breaking changes in 5.0
+
+This section discusses the changes that you should be aware of if you upgrade
+the Beats from version 1.x to 5.x.
+
+==== Configuration file changes
+
+Beats 5.0 come with a more powerful and more robust <<config-file-format,file
+configuration format>>. While this necessarily cause a few breaking changes, we
+provide a migration script that should make upgrading easy. Please see the
+<<upgrading>> section.
+
+For consistency and clarity, all configuration options that are specific to a
+particular Beat are now prefixed by the name of the Beat. For example, the
+`interfaces.device` option from Packetbeat becomes
+`packetbeat.interfaces.device`. This change is made automatically by the
+migration script.
+
+Similarly, common settings available in all Beats are now top level. As part of
+this, the options under the `shipper` section are moved to the top level. For
+example, the `shipper.name` settings becomes `shipper`. This change is made
+automatically by the migration script.
+
+For consistency with the rest of the Elastic stack, several TLS/SSL related
+options where changed:
+
+* the name of the section was renamed from `tls` to `ssl`
+* `certificate_key` was renamed to `key`
+* `insecure` was replaced by `verification_mode`
+* `min_version` and `max_version` were replaced with `supported_protocols`
+
+These changes are made automatically by the migration script.
+
+In Filebeat, we refactored the configuration around when to close and when to
+start ignore files. Options like `ignore_older` and `force_close_older` were
```

Comment:
ignore_older is one of the few options that stayed the same. `close_older` would be a good example.